contracts/src/v0.8/vrf/dev/VRFCoordinatorV2_5.sol

```diff
@@ -179,6 +180,9 @@ contract VRFCoordinatorV2_5 is VRF, SubscriptionAPI, IVRFCoordinatorV2Plus {
     if (fulfillmentFlatFeeNativePPM > 0 && fulfillmentFlatFeeLinkDiscountPPM >= fulfillmentFlatFeeNativePPM) {
       revert LinkDiscountTooHigh(fulfillmentFlatFeeLinkDiscountPPM, fulfillmentFlatFeeNativePPM);
     }
+    if (nativePremiumPercentage > 100 || linkPremiumPercentage > 100) {
+      revert InvalidPremiumPercentage(nativePremiumPercentage, linkPremiumPercentage);
```

Comment:
Not too sure if we need to follow the pattern of emitting the max value in the event.

Reply:
Where did you see this pattern so I can take a look?

Reply:
Here. https://github.com/smartcontractkit/chainlink/pull/12314

Seems like the errors are emitting the allowable values as well.

Reply:
I see - would that mean I have to create two categories of errors, one for native and another one for LINK? Or would just one be enough and you should be able to figure it out based on the `have` and `want` values returned? Consider I don't want to blow up the contract size just because I want to keep things clear.

Reply:
I would say just the max value is enough? And since each premium percentage has the same max value, we could just keep it to a single error.

Reply:
Should we send a string "native" or "link" in the event to identify which premium is over the bounds?

Reply:
We can but I don't think it's needed? Like just stare at the two numbers and see which is over the limit. lol

Reply:
I thought about it at the beginning, but I agree with @leeyikjiun here. Just by looking at the numbers, it should be clear which one is it. No need to increase the contract size with additional event types.